### ISSUE TITLE
`NullPointerException` at `RelayService.goodbye`

### ISSUE THREAD

**Issue by oakkitten** [ubergeek42/weechat-android#486]
Redmi Note 8, Android 9 (SDK 28), 504.apk (4645c45ce0dce195572ae7b0d6389b20b21ddfcb), 13 days ago
total occurrences 60, 1 user

```
java.lang.NullPointerException: 
  at android.app.AlarmManager.cancel (AlarmManager.java:916)
  at com.ubergeek42.WeechatAndroid.service.RelayService.goodbye (RelayService.java:2092)
  at com.ubergeek42.WeechatAndroid.service.RelayService.onStateChanged (RelayService.java:262)
  at com.ubergeek42.weechat.relay.RelayConnection.onStateChanged (RelayConnection.java:74)
  at com.ubergeek42.weechat.relay.connection.AbstractConnection.disconnect (AbstractConnection.java:64)
  at com.ubergeek42.weechat.relay.connection.AbstractConnection.access$300 (AbstractConnection.java:2157)
  at com.ubergeek42.weechat.relay.connection.AbstractConnection$2.run (AbstractConnection.java:83)
  at java.lang.Thread.run (Thread.java:764)
```

**Comment by oakkitten** [comment]
must be the same:

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:3760)
  at android.app.ActivityThread.access$1700 (ActivityThread.java:201)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1711)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loop (Looper.java:201)
  at android.app.ActivityThread.main (ActivityThread.java:6864)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:547)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:873)
Caused by: java.lang.NullPointerException: 
  at android.app.AlarmManager.cancel (AlarmManager.java:916)
  at com.ubergeek42.WeechatAndroid.service.RelayService.goodbye (RelayService.java:2092)
  at com.ubergeek42.WeechatAndroid.service.RelayService.stop (RelayService.java:181)
  at com.ubergeek42.WeechatAndroid.service.RelayService.onStartCommand (RelayService.java:119)
  at android.app.ActivityThread.handleServiceArgs (ActivityThread.java:3738)
```

**Comment by oakkitten** [comment]
must've been fixed 4a1c2ce3f3cbd3e2193e69368a8e074f96e3617e
